src/test/resources/Features/login.feature

```diff
@@ -0,0 +1,9 @@
+Feature: User login
+
+Scenario: User logs in as ACB Admin
+	Given I am on CHPL admin page	
+	When I fill in username
+	And I fill in Password
+	When I click Log in button 
```

Comment:
Should be an "And", not a "When". These scenarios should make sense to a customer as "English".